static/bidder-params/resetdigital.json

```diff
@@ -0,0 +1,8 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Relevant Digital Adapter Params",
+    "description": "A schema which validates params accepted by the Relevant Digital adapter",
+    "type": "object",
+    "properties": {},
```

Comment:
`ExtResetDigital` from `openrtb_ext/imp_resetdigital.go` defines `bid_id` and `imp_id` as imp extension param. Should include these params in `properties`
 
```
package openrtb_ext

type ExtResetDigital struct {
	BidID string `json:"bid_id"`
	ImpID string `json:"imp_id"`
}
```